clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7117,6 +7117,13 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     }
   }
 
+  // The offloading devices do not support RTTI.
```

Comment:
I see that IsHIPDevice has been added to the check here (compared to what was deleted inside CodeGenModule.h). is that expected?

Thanks